.github/workflows/test.yml

```diff
@@ -0,0 +1,59 @@
+name: SROS2 CI
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily
+    - cron:  '0 20 * * *'
+
+jobs:
+  test_latest:
+    runs-on: ubuntu-latest
+    container: osrf/ros2:devel
+    steps:
+    - run: |
+        apt-get -qq update
+        apt-get -qq upgrade -y
+        apt-get -qq install -y curl libasio-dev libtinyxml2-dev
+    - uses: mikaelarguedas/action-ros-ci@sros2-version
```

Comment:
Here we are using a custom version to:
- use colcon provided argument for pytest coverage instead of overriding them. More details at  https://github.com/ros-tooling/action-ros-ci/pull/109
- Avoid having to install pip and the unreleases colcon lcov-result that otherwise if making ci fail https://github.com/ros-tooling/action-ros-ci/commit/f9c55f383bd617b8b39e7346124db1d2b9b9922b

Reply:
Given that it sounds like these issues will be fixed upstream, would you mind making a comment similar to this so we know in the future 1) why we're using a custom version and 2) when it should be revisited?

Reply:
Other than this comment, this looks good to me.

Reply:
added comment at https://github.com/ros2/sros2/pull/178/commits/535a700bb73a831bfb70e0020e5f3a31f44472b6